wrapper/cApiWrapper.cpp

```diff
@@ -88,7 +88,7 @@ ctlInit(
 #ifdef WINDOWS_UWP
             hinstLib = LoadPackagedLibrary(strDLLPath, 0);
 #else
-            hinstLib = LoadLibraryW(strDLLPath);
+            hinstLib = LoadLibraryExW(strDLLPath, NULL, LOAD_LIBRARY_SEARCH_SYSTEM32);
```

Comment:
Thanks for your feedback.
How about using (LOAD_LIBRARY_SEARCH_SYSTEM32 | LOAD_LIBRARY_SEARCH_APPLICATION_DIR)? This will allow easier developer debug work as well as restrict DLL search directories. It helps with developer work as one can easily try newer IGCL dll builds. Let me know your thoughts.

Reply:
Thank you so much for reply and recommendation. Yes - Adding the 'LOAD_LIBRARY_SEARCH_APPLICATION_DIR' will helps us during debug to load the DLL from application directory. I have already made the changes to support this based on your recommendation. Used the load flag -'LOAD_LIBRARY_SEARCH_APPLICATION_DIR' only in debug mode by using the preprocessor #ifdef _DEBUG like following code. In production code we don't want to include this flag because of the security issue , and provide only the specific path to load the DLL(System32). Would you please support to merge this PR request soon . We are waiting for this fix to be included in our production code. Thanks again for your reply.

#ifdef _DEBUG
            dwFlags = LOAD_LIBRARY_SEARCH_SYSTEM32 | LOAD_LIBRARY_SEARCH_APPLICATION_DIR;
#else
            dwFlags = LOAD_LIBRARY_SEARCH_SYSTEM32;

Reply:
Since the wrapper is an auto generated code from an internal repo, I cannot allow a direct merge. Will take your feedback and incorporate the same asap. Will update in this thread itself.

Reply:
Thank you so much for your kind full support. I understand. We are waiting for your update to pickup the changes in our production code . Could you please support soon ,and let me know. 
Looking forward.Thanks in advance.

Reply:
Since migration from internal repo to this public is taking time and you have been waiting, I got approval to manually push just this change alone. Thanks for your understanding & feedback.